processing/src/main/java/org/apache/carbondata/processing/store/CarbonFactDataHandlerModel.java

```diff
@@ -609,5 +616,13 @@ public short getWritingCoresCount() {
   public DataMapWriterListener getDataMapWriterlistener() {
     return dataMapWriterlistener;
   }
+
+  public Boolean[][] getComplexDictionaryFields() {
```

Comment:
change this to boolean[][] getComplexDictionaryFields()

Reply:
Done